src/main/java/bankservice/demo/repository/RoleRepository.java

```diff
@@ -0,0 +1,11 @@
+package bankservice.demo.repository;
+
+import bankservice.demo.entity.Role;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+public interface RoleRepository extends JpaRepository<Role, Long> {
+    @Query("FROM Role WHERE roleName =: rolename")
+    Optional<Role> getByName(Role.RoleName roleName);
```

Comment:
Better name this method getByRoleName or something like this

Reply:
Fixed